web/styles/zulip.css

```diff
@@ -2103,3 +2103,7 @@ body:not(.hide-left-sidebar) {
     font-style: italic;
     margin-right: auto;
 }
+
+#topic-summary-modal {
+    width: 45rem;
```

Comment:
I think `em` would be a better unit for this? Also, what happens at mobile web sizes?

Reply:
@evykassirer Has a PR up that might be relevant: #33270 .

Reply:
At mobile `max-width: calc(100% - 32px);` kicks in. Pushed couple commits to #33270 for changing the unit.